### PR TITLE
transfer : (en-us)home page by zh-cn

### PR DESCRIPTION
**Pull request by lzf971107**
transfer : (en-us)home page by zh-cn

### CODE REVIEW

File: src/components/footer/index.jsx [hunk 1]
```diff
@@ -18,7 +18,6 @@ class Footer extends React.Component {
       <footer className="footer-container">
         <div className="footer-body">
           <img src={getLink(logo)} />
-          <p className="docsite-power">website powered by docsite</p>
```
Comment (lovepoem) [inline src/components/footer/index.jsx]: Why delete this?

Comment (lzf971107) [reply src/components/footer/index.jsx]: I see similar websites do not have this sentence .
I can roll back this submission if necessary.

Comment (lovepoem) [reply src/components/footer/index.jsx]: I think we need to keep it

Comment (lzf971107) [reply src/components/footer/index.jsx]: received

**Pull request merged**